torch/nn/utils/parametrize.py

```diff
@@ -108,7 +112,7 @@ def forward(self) -> Tensor:
         x = self.original
         for module in self:
             x = module(x)
-        if x.size() != self.original.size():
+        if not self.allow_resize and x.size() != self.original.size():
             raise RuntimeError(
                 "The parametrization may not change the size of the parametrized tensor. "
```

Comment:
You might want to update this error to point the user to the existence of this flag to make it discoverable.